### ISSUE TITLE
vrae.transform() discards sequences ?

### ISSUE THREAD

**Issue by flxshk** [tejaslodaya/timeseries-clustering-vae#12]
@tejaslodaya 

Hi, thanks for this handy tool. I run into troubles after training my VRAE instance. When I want to cluster it in order to assign labels to each observation, I get a less output sequences than I put in. 

Any idea what can cause this problem? (I strictly followed your .ipynb during implementation.)

thanks.

```
train_dataset.tensors[0].shape 
>>> torch.Size([82719, 48, 4])

z_run = vrae.transform(train_dataset)

z_run.shape
>>> (82688, 24)

```

**Comment by flxshk** [comment]
closing this wrt. #11
